### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by marminot**
add additional new translations

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,767 @@
...
+	"block.securitycraft.reinforced_spruce_pressure_plate": "Plaque de pression en épicéa blindée",
+	"block.securitycraft.reinforced_birch_pressure_plate": "Plaque de pression en bouleau blindée",
+	"block.securitycraft.reinforced_jungle_pressure_plate": "Plaque de pression en acajou blindée",
+	"block.securitycraft.reinforced_acacia_pressure_plate": "Plaque de pression en acacia blindée",
+	"block.securitycraft.reinforced_dark_oak_pressure_plate": "Plaque de pression en chêne noir blindée",
+	"block.securitycraft.reinforced_red_sand": "Sable rouge blindé",
+	"block.securitycraft.reinforced_bookshelf": "Etagère blindée",
+	"block.securitycraft.reinforced_redstone_block": "Bloc de redstone blindé",
+
+	"item.securitycraft.casseur de code": "Décrypteur",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed the language key here. It should be `item.securitycraft.codebreaker`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 2]
```diff
@@ -0,0 +1,767 @@
...
+	"help.securitycraft.reinforced_stone_pressure_plate.info": "La Plaque de pression en pierre blindée ne peut être activée que par son propriétaire ou des joueurs sur liste blanche.",
+
+"help.securitycraft.codebreaker.info": "Le décrypteur va casser n’importe quel code d’un coffre protégé par mot de passe, ou d’un four protégé par mot de passe en cliquant dessus avec le bouton droit. (5 utilisations avec 2/3 de chances que cela fonctionne)",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: Indent, please.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 3]
```diff
@@ -0,0 +1,767 @@
...
+	"gui.securitycraft:scManual.recipe.limited_use_keycard": "Désactivé",
+	"gui.securitycraft:scManual.intro.1": "Démarrer avec SecurityCraft",
+	"gui.securitycraft:scManual.intro.2": "par Geforce et bl4ckscor3",
+	"gui.securitycraft:scManual.ownableBloc": "Propriétaire: ce bloc résiste aux explosions et ne peut être cassé que par le joueur qui l’a déposé à l’aide du Déplaceur De bloc universel.",
+	"gui.securitycraft:scManual.passwordProtectedBloc": "Protégé par Mot de passe : ce bloc ne peut être utilisé que si un mot de passe défini par le propriétaire est correctement entré.",
+	"gui.securitycraft:scManual.viewActivatedBloc": "Affichage activé: ce bloc peut être activé en le survolant avec votre curseur lorsque vous êtes à portée.",
+	"gui.securitycraft:scManual.explosiveBloc": "Explosif: ce bloc peut être déclenché à distance, et peut être activé / désactivé, à l’aide de l’outil d’accès aux mines à distance.",
+	"gui.securitycraft:scManual.customizableBloc": "Personnalisable: vous pouvez utiliser le Modificateur de Bloc universel et les différents modules ajoutés par SecurityCraft pour personnaliser les fonctions de ce bloc.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` of `Block` at the end of each of these lines' language key.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 4]
```diff
@@ -0,0 +1,767 @@
...
+	"gui.securitycraft:scManual.recipe.adminTool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration.",
+	"gui.securitycraft:scManual.recipe.admin_tool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration..",
+	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Fabriqué à partir de Bloc de Cristal de Quartz ordinaire dans un Blindeur de blocs universel.",
+	"gui.securitycraft:scManual.recipe.sc_manual": "Le manuel SecurityCraft est uniquement disponible via l’inventaire ou en tapant \"/sc help\" (sans apostrophes).",
+	"gui.securitycraft:scManual.reinforced": "Blocs blindés",
+	"gui.securitycraft:scManual.designedBy": "Conçu par: %s",
+	"gui.securitycraft:ims.hostileAndPlayers": "Mobs hostiles & joueurs",
+	"gui.securitycraft:ims.hostile": "Mobs hostiles",
+	"gui.securitycraft:ims.target": "Cible:",
+	"gui.securitycraft:invScan.emitredstone": "Emettre redstone",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed the casing of the last part of the language keys here. It should be `emitRedstone`, not `emitredstone`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 5]
```diff
@@ -0,0 +1,767 @@
...
+	"gui.securitycraft:invScan.checkInv": "Vérifier l’inventaire",
+	"gui.securitycraft:invScan.explanation.1": "Ce paramètre vérifiera l’inventaire",
+	"gui.securitycraft:invScan.explanation.2": "d’un joueur, et, s’il contient un",
+	"gui.securitycraft:invScan.explanation.checkInv.3": "élément interdit, l’effaçera",
+	"gui.securitycraft:invScan.explanation.checkInv.4": "de l’inventaire.",
+	"gui.securitycraft:invScan.explanation.emitredstone.3": "élément interdit, émission d’un signal ",
+	"gui.securitycraft:invScan.explanation.emitredstone.4": "redstone pendant 3 secondes.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed the casing of the last part of the language keys here. It should be `emitRedstone`, not `emitredstone`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 6]
```diff
@@ -0,0 +1,767 @@
...
+	"gui.securitycraft:keycardSetup.save": "enregistrer & continuer",
+	"gui.securitycraft:keycardSetup.explanation.1": "Installation de Carte d‘accès",
+	"gui.securitycraft:keycardSetup.explanation.2": "Sélectionner le niveau de sécurité",
+	"gui.securitycraft:keycardSetup.explanation.3": "voulu. Seuls les joueurs avec une",
+	"gui.securitycraft:keycardSetup.explanation.4": "carte d’accès ayant le bon ",
+	"gui.securitycraft:keycardSetup.explanation.5": "niveau de sécurité pourront ",
+	"gui.securitycraft:keycardSetup.explanation.6": "utiliser ce lecteur de",
+	"gui.securitycraft:keycardSetup.explanation.7": "cartes.",
+	"gui.securitycraft:keycardSetup.lvlNeeded": "Niveau de sécurité requis:",
+	"gui.securitycraft:protectedFour.name": "Four Protégé",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed the language key again. `protectedFour` should be `protectedFurnace`

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 7]
```diff
@@ -0,0 +1,767 @@
...
+	"gui.securitycraft:srat.notBound": "Non associé!",
+	"gui.securitycraft:srat.sentryLocations": "Sentinelle à #location",
+	"gui.securitycraft:srat.name": "Outil d’accès à distance pour les sentinelles",
+	"gui.securitycraft:srat.modifyAll": "Modifier toutes les sentinelles associées",
+	"gui.securitycraft:password.invalidCode": "Code Invalide!",
+	"gui.securitycraft:password.setup": "Installation",
+	"gui.securitycraft:camera.activateNightVision": "Activer la vision nocturne",
+	"gui.securitycraft:camera.exit": "Exit Caméra",
+	"gui.securitycraft:camera.toggleredstone": "Basculer le signal redstone",
+	"gui.securitycraft:camera.toggleredstoneNote": "(nécessite un module redstone)",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed the casing of the last part of the language keys here. It should be `toggleRedstone`/`toggleRedstoneNote`, not `toggleRedstone`/`toggleredstoneNote`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 8]
```diff
@@ -0,0 +1,767 @@
...
+	"gui.securitycraft:camera.zoom": "Zoomer/dézoomer",
+	"gui.securitycraft:monitor.selectCaméras": "Sélectionner la camera à voir:",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed `e` to `é` in the language key here.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 9]
```diff
@@ -0,0 +1,767 @@
...
+	"gui.securitycraft:editModule.remove": "Supprimer",
+	"gui.securitycraft:editModule.copy": "Copier la Liste",
+	"gui.securitycraft:editModule.paste": "Coller la Liste",
+	"gui.securitycraft:editModule.clear": "Effacer la Liste",
+	"gui.securitycraft:blockPocketManager.activate": "Activer",
+	"gui.securitycraft:blockPocketManager.deactivate": "Désactiver",
+	"gui.securitycraft:blockPocketManager.size": "Taille: %sx%sx%s",
+
+	"messages.securitycraft:bugReport": "Vous pouvez signaler un plantage ou un bug en utilisant ce formulaire GoogleDocs",
+	"messages.securitycraft:securityCaméra.mounted": "Vous êtes maintenant connectés sur une caméra. Utilisez les touches WASD pour déplacer la vue de la caméra et les boutons +/- pour zoomer et dézoomer.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed `e` to `é` in the language key here.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 10]
```diff
@@ -0,0 +1,767 @@
...
+	"messages.securitycraft:retinalScanner.hello": "Bonjour #.",
+	"messages.securitycraft:retinalScanner.notOwner": "Je suis désolé, vous ne pouvez pas utiliser ce scanner. Il appartient à #.",
+	"messages.securitycraft:portableRadar.withName": "#p est près de votre radar portable nommé #n.",
+	"messages.securitycraft:portableRadar.withoutName": "#p est près de votre radar portable à #l.",
+	"messages.securitycraft:casseur de codeDisabled": "Le décrypteur a été désactivé via le fichier de configuration.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed `codebreaker` to `casseur de code` in the language key here. It should be `messages.securitycraft:codebreakerDisabled`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 11]
```diff
@@ -0,0 +1,767 @@
...
+	"messages.securitycraft:cameraMonitor.rightclickToView": "Faites un clic droit sur une caméra de sécurité pour la voir.",
+	"messages.securitycraft:mrat.noSlots": "Il n’y a plus d’emplacements vides pour associer cette mine!",
+	"messages.securitycraft:mrat.cantBind": "Vous ne pouvez pas associer une mine qui ne vous appartient pas.",
+	"messages.securitycraft:mrat.bound": "Association de la mine à # à l’outil d’accès à distance.",
+	"messages.securitycraft:mrat.unbound": "Dissociation de la mine à # de l’outil d’accès à distance.",
+	"messages.securitycraft:srat.noSlots": "Il n’y a plus d’emplacements vides pour associer cette sentinelle !",
+	"messages.securitycraft:srat.cantBind": "Vous ne pouvez pas associer une sentinelle qui ne vous appartient pas.",
+	"messages.securitycraft:srat.bound": "Association de la sentinelle à # à l’outil d’accès à distance.",
+	"messages.securitycraft:srat.unbound": "issociation de la sentinelle à # de l’outil d’accès à distance.",
+	"messages.securitycraft:universalBlocModifier.cantModify": "\"",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` from `Block` in the language key here. It should be `messages.securitycraft:universalBlockModifier.cantModify`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 12]
```diff
@@ -0,0 +1,767 @@
...
+	"messages.securitycraft:tip.trello": "Consultez le tableau Trello pour SecurityCraft et signalez des bogues ou faites-nous des suggestions!",
+	"messages.securitycraft:tip.patreon": "Voulez-vous prendre en charge SecurityCraft?",
+	"messages.securitycraft:tip.discord": "Rejoignez notre serveur Discord pour discuter et recevoir des mises à jour et des annonces importantes concernant SecurityCraft!",
+	"messages.securitycraft:tip.scserver": "Le serveur officiel SecurityCraft est de retour ! Rejoignez-nous sur cette IP: 64.38.249.237",
+	"messages.securitycraft:universalKeyChanger.notKeypad": "Clic droit sur un clavier avec le changeur de code universel pour changer son mot de passe.",
+	"messages.securitycraft:universalKeyChanger.passcodeChanged": "Le code a été changé.",
+	"messages.securitycraft:passwordProtected.notSetUp": "Ce bloc n’a pas encore de mot de passe. Vous ne pouvez pas interagir avec.",
+	"messages.securitycraft:sentry.needsBlocBelow": "Les Sentinelles se placent sur un bloc!",
+	"messages.securitycraft:sentry.needsBlocsAround": "Assurez-vous que votre Sentinelle est entourée de blocs sur ses quatre côtés!",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` from `Block` in the language keys here. They should be `messages.securitycraft:sentry.needsBlockBelow` and `messages.securitycraft:sentry.needsBlocksAround`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 13]
```diff
@@ -0,0 +1,767 @@
...
+	"messages.securitycraft:sentry.mode1": "Mode actuel: agressif (toujours actif, attaque les mobs et les joueurs hostiles)",
+	"messages.securitycraft:sentry.mode2": "Mode actuel: Camouflage (Actif lorsqu’un joueur s’approche, attaque les joueurs) ",
+	"messages.securitycraft:sentry.mode3": "Mode actuel: inactif (jamais actif, n’attaque rien)",
+	"messages.securitycraft:blockpocket.multipleManagers": "Il y a plus d’un gestionnaire de Bloc de poche dans la structure.",
+	"messages.securitycraft:blockpocket.invalidBloc": "Bloc invalide \"%2$s\" à la position %1$s!",
+	"messages.securitycraft:blockpocket.invalidBloc.pillar": "Bloc invalide \"%2$s\" à la position %1$s! (Assurez-vous que le pilier tourne correctement!)",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` from `Block` in these language keys. They should be `messages.securitycraft:blockpocket.invalidBlock` and `messages.securitycraft:blockpocket.invalidBlock.pillar`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 14]
```diff
@@ -0,0 +1,767 @@
...
+	"messages.securitycraft:blockpocket.unowned": "Vous ne possédez pas le bloc \"%2$s\" à la position %1$s!",
+	"messages.securitycraft:blockpocket.activated": "Bloc de poche activé!",
+	"messages.securitycraft:blockpocket.deactivated": "Bloc de poche désactivé!",
+
+	"jei.securitycraft:reinforcedBloc.info": "Clic droit sur un Blindeur de blocs universel et placez le bloc suivant dans l’emplacement. Sinon, vous pouvez placer le bloc suivant vers le bas et le casser avec le Blindeur de blocs universel.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` from `Block` in the language key here.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 15]
```diff
@@ -0,0 +1,767 @@
...
+	"tooltip.securitycraft:module.itemAddons.added": "Ajouté",
+	"tooltip.securitycraft:keycard.uses": "utilisations restantes:",
+
+	"itemGroup.securitycraft.decoration": "SecurityCraft: Décoration",
+	"itemGroup.securitycraft.explosives": "SecurityCraft: Explosifs",
+	"itemGroup.securitycraft.technical": "SecurityCraft: Technique",
+
+	"module.securitycraft.ims.whitelist_module.description": "L’ajout d’un module de liste blanche à un S.M.I. permettra aux joueurs inscrits sur la liste blanche de pénétrer dans la portée du I.M.S sans être ciblés.",
+	"module.securitycraft.inventory_scanner.whitelist_module.description": "L’ajout d’un module de liste blanche à un scanner d’inventaire permettra aux joueurs sur liste blanche de parcourir le champ de scan sans que leur inventaire soit contrôlé.",
+	"module.securitycraft.inventory_scanner.smart_module.description": "L’ajout d’un module intelligent à un scanner d’inventaire permettra de contrôler les enchantements et les autres changements de métadonnées lors de la numérisation des
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed `Keycard` to `Carte d‘accès` in the language keys here.

Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: LookingGlass is a mod name and should not be translated. Please change `Miroir` back to `LookingGlass`.

Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` from `Block` in the language keys here.

Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` from `Block` in the language keys here.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 16]
```diff
@@ -0,0 +1,767 @@
...
+	"help.securitycraft.reinforced_iron_bars.info": "Les Barreaux en fer renforcés sont les mêmes que les barreaux de fer standards, sauf que seule la personne qui les a déposé peut la détruire à l’aide du déplaceur de bloc universel.",
+	"help.securitycraft.electrified_iron_fence.info": "La Barrière en fer électrifiée est la même que la Barrière standard, sauf qu’elle est indestructible, et blessera tous ceux qui la toucheront, sauf celui qui l‘a posée.",
+	"help.securitycraft.ims.info": "Le S.M.I, ou Système de Munitions Intelligent, est une mine à quatre utilisations qui ciblera les joueurs (ou mobs) qui entrent dans son rayon de tir. Une fois qu’une mine est lancée, elle traquera l’entité et explosera. Les mines peuvent être rechargées en cliquant avec le bouton droit sur S.M.I. avec une mine rebondissante. Une fois les quatre mines utilisées, le S.M.I. va s’autodétruire.",
+	"help.securitycraft.protecto.info": "Le protecto, inspiré par l’ennemi du jeu \"Megaman: Battle Network 2 \" de Capcom, attaquera tout mob dans un rayon de 10 blocs avec des éclairs toutes les 10 secondes. Il doit pleuvoir et le ciel doit être visible pour que ça marche.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: Remove the space in front of the second `\`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 17]
```diff
@@ -0,0 +1,767 @@
...
+	"help.securitycraft.motion_activated_light.info": "La lumière à détecteur de mouvement émettra une lumière, semblable à une lampe redstone, lorsqu’un joueur se trouve à une certaine portée du bloc (configurable).",
+	"help.securitycraft.trophy_system.info": "inspiré par le gadget vu pour la première fois dans \"Call of Duty: Modern Warfare 3 \" publié par Activision, cible les flèches et les boules de feu ennemies et les détruira dans la portée après un temps de charge d’une demi-seconde",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: Remove the space in front of the second `\`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 18]
```diff
@@ -0,0 +1,767 @@
...
+	"help.securitycraft.block_pocket_manager.info": "Le Gestionnaire de Bloc de poche est utilisé pour gérer le Bloc de poche, qui est une pièce dans laquelle seul son propriétaire et les joueurs sur liste blanche peuvent entrer. Lorsque vous ouvrez son interface graphique, vous voyez une sélection de tailles. Choisissez-en une, puis construisez le Bloc de poche autour du gestionnaire. Assurez-vous que le gestionnaire est placé à l’extérieur de la couche inférieure. Le Bloc de poche est un cube de la taille que vous avez choisie. Les coins sont construits avec du cristal de Quartz sculpté blindé, les blocs entre les coins sont construits avec des piliers de cristal de Quartz blindé correctement orientés, et les sol / mur / plafond sont construits avec des murs de Bloc de poche. Voir cette image pour un exemple de son apparence: https://i.imgur.com/opc7dqO.png - Cliquer sur \"Activer \" après la construction de la structure activera tous les modules ajoutés. Voir l’interface graphique de personnalisation pour plus d’informations (clic droit avec ModificateUr de Bloc universel).",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: Remove the space in front of the second `\`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 19]
```diff
@@ -0,0 +1,767 @@
...
+	"help.securitycraft.remote_access_sentry.info": "L’outil d’accès à distance aux sentinelles vous permettra de contrôler les sentinelles à distance. Cliquez avec le bouton droit sur une sentinelle pour l‘associer à l’outil. Cliquez avec le bouton droit dans l’air (avec l’outil équipé) pour ouvrir l’interface graphique de l’outil, ce qui vous permettra de changer le mode de chaque sentinelle associée, ou de toutes les sentinelles associées en même temps",
+	"help.securitycraft.bucket_f_water.info": "La fausse eau agit comme l’eau, attendez-vous à ce qu’elle vous fasse mal au toucher.",
+	"help.securitycraft.bucket_f_lava.info": "La fausse lave agit comme la lave, sauf qu’elle vous guérit au lieu de vous blesser.",
+	"help.securitycraft.universal_block_modifier.info": "Le modificateur de bloc universel est utilisé pour personnaliser de nombreux ajouts par SecurityCraft. Cliquez avec le bouton droit sur un bloc pour ouvrir l’interface graphique de personnalisation. Vous pouvez ensuite insérer des \"modules \" pour personnaliser le fonctionnement de ce bloc.",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: Remove the space in front of the second `\`.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 15 above)
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: `\\ n \\ n` should be changed to `\\n\\n` and the space infront of the second `\` of `\"false \"` should be removed as well.

Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: Remove the space in front of the second and fourth `\`.

**Comment by marminot** [comment]
try to do what you wanted, I am not used to git, i am only 10

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 18 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: There's also a capital `U` at the end in `ModificateUr` which I'm assuming should not be there.

**Comment by bl4ckscor3** [comment]
Nothing changed. You can just edit the files again at the following links, change what I commented at (some comments are referencing multiple lines) and click the green "Commit changes" button below once you're done.

https://github.com/marminot/SecurityCraft/blob/1.15.x/src/main/resources/assets/securitycraft/lang/fr_fr.json
https://github.com/marminot/SecurityCraft/blob/1.15.x/src/main/resources/assets/securitycraft/lang/en_us.json

**Comment by bl4ckscor3** [comment]
I forgot about this: Please translate `"gui.securitycraft:scManual.author": "Translated by marminot"` as well, that will show up in the SecurityCraft Manual when the user is using your translation.

**Comment by marminot** [comment]
is it oK now?

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 20]
```diff
@@ -0,0 +1,768 @@
...
+	"tooltip.securitycraft:module.itemAddons.added": "Ajouté",
+	"tooltip.securitycraft:keycard.uses": "utilisations restantes:",
+
+	"itemGroup.securitycraft.decoration": "SecurityCraft: Décoration",
+	"itemGroup.securitycraft.explosives": "SecurityCraft: Explosifs",
+	"itemGroup.securitycraft.technical": "SecurityCraft: Technique",
+
+	"module.securitycraft.ims.whitelist_module.description": "L’ajout d’un module de liste blanche à un S.M.I. permettra aux joueurs inscrits sur la liste blanche de pénétrer dans la portée du I.M.S sans être ciblés.",
+	"module.securitycraft.inventory_scanner.whitelist_module.description": "L’ajout d’un module de liste blanche à un scanner d’inventaire permettra aux joueurs sur liste blanche de parcourir le champ de scan sans que leur inventaire soit contrôlé.",
+	"module.securitycraft.inventory_scanner.smart_module.description": "L’ajout d’un module intelligent à un scanner d’inventaire permettra de contrôler les enchantements et l
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally changed `Keycard` to `Carte d‘accès` in the language keys here.

Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: LookingGlass is a mod name and should not be translated. Please change `Miroir` back to `LookingGlass`.

Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json]: You accidentally removed the `k` from `Block` in the language keys here.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 15 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

**Comment by bl4ckscor3** [comment]
There are still some things left to do over the whole file.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 5 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 1 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 6 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 7 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 8 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 9 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 10 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 11 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 12 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 13 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json (hunk 14 above)
Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/fr_fr.json]: Still needs to be changed.

**Comment by JoyBates** [comment]
So uh, here’s a file Geforce sent to me and TheCrazyFreezee back in November. We worked on it until January 18. There’s nothing wrong with marminot’s translations, but considering I redid the English manual at the same time, the translations in our file are more accurate. ca11 did add a few items that weren’t in the original English file at the time, however. Proceed how you wish.

**Pull request merged**